cli/install/install.go

```diff
@@ -391,11 +391,14 @@ func installTt(version string, binDir string, flags InstallationFlag, distfiles
 	}
 
 	// Check that the version exists.
+	// The tag format in tt is vX.Y.Z, but the user can use the X.Y.Z format
```

Comment:
Should we add a note to the change log about this?

Reply:
Good question. @LeonidVas what do you think?

Reply:
I don't think this is necessary because the user will not see these changes.